src/Symfony/Component/Security/Core/Encoder/SodiumPasswordEncoder.php

```diff
@@ -85,6 +85,10 @@ public function isPasswordValid(string $encoded, string $raw, ?string $salt): bo
             return password_verify($raw, $encoded);
         }
 
+        if (0 !== strpos($encoded, '$argon')) {
```

Comment:
this currently bypasses the check above about the length

Reply:
The 72 character limit for bcrypt?  SHA256/512 doesn't have that particular limit (at least as implemented in CentOS)

Reply:
Yes sure, what I mean is that we will now call the function for bcrypt too.